spectral-cube/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = 'spectral-cube' %}
+{% set version = GIT_DESCRIBE_TAG ~ ".dev" ~ GIT_DESCRIBE_NUMBER ~ "+" ~ GIT_DESCRIBE_HASH %}
+{% if version[0] == 'v' %}
+{%   set version = version[1:] %}
+{% endif %}
+{% set number = '1' %}
+
+about:
+    home: https://github.com/radio-astro-tools/{{ name }}
+    license: BSD
+    summary: Library for reading and analyzing astrophysical spectral data cubes
+
+build:
+    number: {{ number }}
+
+package:
+    name: {{ name }}
+    version: {{ version }}
+
+requirements:
+    build:
+    - astropy
+    - numpy >=1.8.0
+    - radio_beam
```

Comment:
There does not appear to be a `radio_beam` package in the conda defaults channel, nor is there a package distributed as part of astroconda. Is the intention for a new recipe for `radio_beam` to be added as well?

Reply:
No it's not, sorry this should be a `pip` requirement. Is there a way to express that in the recipe?